envoy/http/alternate_protocols_cache.h

```diff
@@ -128,9 +144,17 @@ class AlternateProtocolsCache {
    * @return the number if entries in the map.
    */
   virtual size_t size() const PURE;
+
+  /**
+   * @param origin The origin to get HTTP/3 status for.
+   * @return the status tracker if there is one. Otherwise, nullptr.
+   */
+  virtual std::shared_ptr<AlternateProtocolsCache::Http3StatusTracker>
+  getHttp3StatusTracker(const Origin& origin) PURE;
 };
 
 using AlternateProtocolsCacheSharedPtr = std::shared_ptr<AlternateProtocolsCache>;
+using Http3StatusTrackerSharedPtr = std::shared_ptr<AlternateProtocolsCache::Http3StatusTracker>;
```

Comment:
nit: can we use this type on line 152?

Reply:
Line 152 is in class body, and doesn't have this alias declared yet.